### PR TITLE
fix: broken PR template links and version bump

### PR DESCRIPTION
**Pull request by robrap**
**TL;DR -**

* The links in the PR template for release process,
setup.py, and package.json have been fixed. For
setup.py and package.json, also added a line number
link using a commit hash.
*  Updating the version because it was determined that the
following unreleased PR warrants a minor version bump: #1250

**Developer Checklist**

- [X] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] ~~Translations and JS/SASS compiled~~
- [X] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)
 
 **Developer Checklist**
 
-- [ ] Reviewed the [release process](./release_process.md)
+- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/docs/release_process.rst)
```
Comment (nsprenkle) [inline .github/pull_request_template.md:13]: This changes the target, should be:

```suggestion
- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
```

**Review by jnlapierre — APPROVED** [review]
LGTM

**Review by nsprenkle — APPROVED** [review]
:shipit:
